spec/queries/customer_invoices_query_spec.rb

```diff
@@ -0,0 +1,160 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe InvoicesQuery, type: :query do
```

Comment:
The name of the query should match the name of the spec file. Could you also merge this test case into the existing `spec/queries/invoice_query_spec.rb`?

Reply:
Good point, done !